packages/xod-client/src/user/actions.js

```diff
@@ -0,0 +1,18 @@
+import url from 'url';
+import { UPDATE_COMPILE_LIMIT } from './actionTypes';
+import { ORIGIN_COMPILE } from './constants';
+
+export const updateCompileLimit = () => (dispatch) => {
+  fetch(url.format({
+    ...url.parse(process
+      ? process.env.ORIGIN_COMPILE || ORIGIN_COMPILE
```

Comment:
Unnecessary check for `process` variable... see `webpack.config.js` and `xod-client/src/utils/siteLinks.js`